src/F0_output.py

```diff
@@ -14,6 +14,8 @@
 from src.constants_json_strings import (
     KPI,
     OPTIMIZED_FLOWS,
+    DEMANDS,
+    SUPPLIES,
```

Comment:
I think instead of `SUPPLIES` `RESOURCES` would be a better choice (if I am correct that this plots PV and stuff).

Reply:
Ok

Reply:
Done in cdf2888fbb2dab98fc3b730ec8400560acad262d